item.rb

```diff
@@ -26,7 +26,9 @@ def add_label(label)
   end
 
   def can_be_archived?
-    true if (DateTime.now - @publish_date).to_i / 365 > 10
+    return true if (Date.today - Date.parse(@publish_date)).to_i / 365 >= 10
```

Comment:
- [OPTIONAL] It is unnecessary to use the return statement when it is already returning the statement by default. This makes your code look simple.

Reply:
@Johnadibe Yeah, I was using return, but rubocop linters removed it xD